realsense_camera/launch/r200_nodelet_modify_params.launch

```diff
@@ -9,6 +9,8 @@
   <!-- These 'arg' tags are just place-holders for passing values from test files.
   The recommended way is to pass the values directly into the 'param' tags. -->
   <arg name="enable_depth"      default="true" />
+  <arg name="enable_ir"         default="true" />
```

Comment:
I don't think they belong in the rgbd launch file. AFAIK they aren't used by RGBD, and they default to false in the header file.